test/unit/delete-stale-entries.spec.ts

```diff
@@ -0,0 +1,21 @@
+import deleteStaleEntries from '../integration/tasks/delete-stale-entries'
+import entryMocks from '../mocks/entries'
+import { expect } from '../helpers'
+import { PlainClientAPI } from 'contentful-management'
+
+describe('DeleteStaleEntries', () => {
+  let plainClient: PlainClientAPI
+  beforeEach(() => {
+    plainClient = {
+      entry: {
+        getMany: () => Promise.resolve(entryMocks),
+        delete: () => Promise.resolve(),
+      },
+    } as unknown as PlainClientAPI
+  })
+
+  it('should delete an entry that is older than one day', async () => {
+    const expectedResult = ['b4ltnOQh2HkAFBZVMzdz7']
+    expect(await deleteStaleEntries(plainClient)).to.eql(expectedResult)
+  })
```

Comment:
I would also check we're not deleting currently used entities here

Reply:
If an entry is older than two hours, we don't use it anymore. Same as for entries. We just copy that entry from `test-base`, run the tests, done